### PR TITLE
Add support for configuring a WP Multi-Network install from a blueprint file

### PR DESCRIPTION
**Pull request by meitar**
This documents the changes introduced by PR #266, as well as enhancing the `vv` script to support WP Multi-Network installation.

A WP Multi-Network is a single WordPress installation that supports multiple Multisite networks. It uses the WP Multi-Network plugin and automatically configures an arbitrary number of additional root domains in addition to the domain specified during invocation of `vv create`.

### CODE REVIEW

**Comment by bradp** [comment]
yooo @meitar want to update this to match the new readme format? thanks ❤️

**Comment by meitar** [comment]
@bradp Sure, IIUC, this is just a matter of including the Multi-Network blueprint docs section in the README's table of contents, yes? If so, it's done!

File: vv [hunk 1]
```diff
@@ -1093,6 +1105,11 @@ PHP"
 	nginx_domain_text="$domain"
 	if [[ $multisite = 'y' ]] && [[ $ms_type = 'subdomain' ]]; then
 		nginx_domain_text="$domain *.$domain"
+		if [ ! -z "$subnetwork_domains" ]; then
+			for subnetwork_domain in $subnetwork_domains; do
+				nginx_domain_text="$nginx_domain_text $subnetwork_domain *.$subnetwork_domain"
```
Comment (bradp) [inline vv]: Should we also create the xip.io domain for this?

Comment (meitar) [inline vv]: Good catch, probably we should. Admittedly, xip.io is not something I've made extensive use of so I neglected this. I see you merged this, but I'll be happy to send you a new PR if you want to see Multi-Network come with xip.io out of the box?

**Pull request merged**